backend/lib/azimutt_web/admin/user/_users_table.html.heex

```diff
@@ -0,0 +1,36 @@
+<div class="flex flex-col border-l border-1 border-slate-800 pl-4 pb-8">
+<div class="text-2xl font-bold text-slate-100 py-2"> <%= assigns.users |> Enum.count() %> users</div>
+    <div class="min-w-full overflow-hidden overflow-x-auto align-middle shadow sm:rounded-sm">
+        <table class="w-full">
+            <thead class="text-gray-100 bg-slate-800 text-xs font-semibold">
+            <tr>
+                <th class="px-1 py-1 text-left" scope="col">Created at</th>
+                <th class="px-1 py-1 text-left" scope="col">Name</th>
+                <th class="px-1 py-1 text-left" scope="col">Email</th>
+                <th class="px-1 py-1 text-left" scope="col">Description</th>
```

Comment:
The last event date would be interesting, but not sure if easy.
Also miss some links ;)